docfiles/docs.js

```diff
@@ -189,14 +189,58 @@ function setupSemantic() {
         }).appendTo(outer);
     });
 
-    $('#printbtn').on("click", function () {
-        window.print();
-    })
+    $('#printbtn').dropdown({
+        action: function(text, value) {
+           
+            // Blocky selectors
+            var blocklyBlockBackground = document.querySelectorAll('.blocklyBlockBackground');
+            var blocklyPath = document.querySelectorAll('.blocklyPath');
+            var blocklyLedOff = document.querySelectorAll('.blocklyLedOff');
+            var blocklyDropdownText = document.querySelectorAll('.blocklyDropdownText');
+            var blocklyText = document.querySelectorAll('.blocklyText');
+            var blocklyLedOn = document.querySelectorAll('.blocklyLedOn');
+            var uiimage = document.querySelectorAll('.ui.image');
+            
+            var collections = [ blocklyBlockBackground, blocklyPath, blocklyLedOff, blocklyDropdownText, blocklyText, blocklyLedOn, uiimage]
+
+            switch (value) {
+                case 'color': 
+                    collections.forEach(function(elements) {
+                        if (elements.length != 0){
+                            addRemovePrintColorClass(elements, 'add');
+                        }
+                    });                       
+                    break;
+                case 'outlines':
+                    collections.forEach(function(elements) {
+                        if (elements.length != 0){
+                            addRemovePrintColorClass(elements, 'remove');
+                        }
+                    });
+                    break;
+                default:
+                    console.log('print');
+           } 
+           
+           window.print();
+           $('#printbtn').dropdown("hide");
+        }
+      });
 
     if (/browsers$/i.test(window.location.href))
         $('.ui.footer').append($('<div class="ui center aligned small container"/>').text('user agent: ' + navigator.userAgent))
 }
 
+function addRemovePrintColorClass(elements,operation) {
+    [].forEach.call(elements, function(el){
+        if (operation === 'add'){
+            el.classList.add("print-color");
```

Comment:
is classList available in IE?

Reply:
It is partially available, but it supports "add" and "remove". https://caniuse.com/#feat=classlist 
I tested it locally :)